io_scene_nif/operators/common_op.py

```diff
@@ -95,3 +82,51 @@ class NifOperatorCommon:
         default=0.0005,
         min=0.0, max=1.0, precision=5,
         options={'HIDDEN'})
+
+
+class CommonImportScale:
+    # Number of nif units per blender unit.
+    scale_correction_import: bpy.props.FloatProperty(
```

Comment:
Let's join this together into one scale correction and apply the inverse on export.

Reply:
That makes sense.
I initially had this as a scene level attribute, but then reverted it https://github.com/niftools/blender_nif_plugin/pull/366/commits/15ea40767276a1bb766b138b6b52444734eddc65 having looked at some other operators.

Also what do you think of these "common" classes, not sure if they are overkill.

Reply:
I think scene prop is better so that export automatically gets the correct value that was used to import the nif, especially for games that don't use the default scale correction.

Reply:
I am just wondering how best to represent the value so that it makes sense to the user then.

Reply:
1 Blender unit = [X Nif units]
where the stuff in brackets is the float entry field